dbld/images/centos7/Dockerfile

```diff
@@ -3,11 +3,13 @@ MAINTAINER Andras Mitzki <andras.mitzki@balabit.com>
 
 RUN yum install -y wget epel-release
 
-RUN cd /etc/yum.repos.d && wget https://copr.fedorainfracloud.org/coprs/czanik/syslog-ng39/repo/epel-7/czanik-syslog-ng39-epel-7.repo
+RUN cd /etc/yum.repos.d && wget https://copr.fedorainfracloud.org/coprs/czanik/syslog-ng311/repo/epel-7/czanik-syslog-ng311-epel-7.repo
 
 ADD dev-dependencies.txt .
 RUN cat dev-dependencies.txt | grep -v "#" | xargs yum -y  install
-
+ENV GRADLE_HOME=/opt/gradle
```

Comment:
I disagree :)

I think this is a packaging issue (@czanik: it is installed from your repo, is it possible to fix this issue there?).

Of course we can fix it, but well... this is not a required step on other distros and this is a workaround, it fixes the packaging in the Dockerfile.
(and just a note: on centos7 we are installing gradle 2.6 not 3.3  - and the installation notes are for the case when you install it manually not from a package).

Alternative solution: download gradle binaries from here: https://gradle.org/releases/ and set `GRADLE_HOME` and other PATH.

Dockerfile is also a documentation of a development environment. I think we should keep them as clean as possible.
This solution is a mixture of manual and package installation in case of gradle which is well... not the best.
IMHO when someone installs gradle(or other tool) from a package then no additional steps are required from the user (setting PATH and GRADLE_HOME).
What do you think?

In case of Debian/Ubuntu it's working. I also checked Fedora 26: on that system it is working as expected.

So my suggestion is to fix the gradle packaging, or, install gradle from binaries (provide an `install_gradle.sh` which downloads the binaries, install it and setup the environment).